src/api/payment-requests/legacy-payment-requests.md

```diff
@@ -75,6 +75,7 @@ See [Asset Types][] for the list of possible `asset` values for each Asset Type.
 | 403    | {% break _ MERCHANT_CONFIGURATION_NOT_FOUND %} | There was no merchant configuration found for the supplied merchantId and clientId.              |
 | 403    | {% break _ NO_AVAILABLE_PAYMENT_OPTIONS %}     | `asset` is not supported by any of the [Asset Types][] that the [Merchant][] is configured with. |
 | 403    | {% break _ INSUFFICIENT_ASSET_VALUE %}         | The patron does not have enough funds to quick pay the payment request.                          |
+| 403    | {% break _ REDIRECT_URL_INVALID %}             | The supplied redirectUrl does not start with one of the `allowedRedirectUrls` on the [Merchant Config][]. |
```

Comment:
We weren't documenting this error code in the legacy route at all so I've added it here too

Reply:
You can't create a legacy payment request with a redirectUrl

Reply:
Do we allow redirectUrls in the legacy route?